src/mono/wasm/runtime/globals.ts

```diff
@@ -59,6 +59,8 @@ export function setRuntimeGlobals(globalObjects: GlobalObjects) {
         beforeOnRuntimeInitialized: createPromiseController<void>(),
         afterOnRuntimeInitialized: createPromiseController<void>(),
         afterPostRun: createPromiseController<void>(),
+        hasDebuggingEnabled: loaderHelpers.hasDebuggingEnabled,
```

Comment:
It should not be duplicated here, instead the other code should access it on `loaderHelpers`

Reply:
Also note the value will not propagate change this way.

Reply:
It is a function